README.md

```diff
@@ -108,6 +108,8 @@ Note: The `react` package depends on the `api` package. After making changes to
 
 Similarly, the `api` package depends on the `auth` package. After changes to `auth`, rebuild it with `yarn build` in `package/auth`, and then restart the `api` development environment.
 
+Make sure the "Enable CORS" option is ON. It can be found in Administration > Settings > General > REST API > Enable CORS. 
```

Comment:
Keep your line just below this line: "Storybook should now be operational." Experiment with `EmbeddedChat` and its components, observing real-time changes in Storybook. 

This will make it easier to read. Additionally add similar line like, please note that it has to enabled to access the functionality of the app. 

Just my suggestions.

Reply:
Also, please add that the 'option' is ON on the Rocket Chat Server, making it clear where exactly the CORS has to be enabled.

Reply:
Done. Lmk if its fine

Reply:
You can also add this line if you want : Please note that it has to enabled to access the functionality of the app.

Because the functionality doesn't work when it is not enabled. User will be able to set it the project up even without enabling it